openvidu-server/src/main/resources/META-INF/additional-spring-configuration-metadata.json

```diff
@@ -159,6 +159,12 @@
 			"description": "Defines if transcoding is allowed or not when OPENVIDU_STREAMS_FORCED_VIDEO_CODEC is not a compatible codec with the browser/client.",
 			"defaultValue": false
 		},
+		{
+			"name": "OPENVIDU_STREAMS_ALLOW_SIMULCAST",
+			"type": "java.lang.Boolean",
+			"description": "Enables usage of Simulcast in those Publishers that support the feature.",
+			"defaultValue": true
+		},
```

Comment:
Is this needed at all? I don't know what is done by this meta-inf file, so I blindly added the param here too...